sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -1115,15 +1124,25 @@
                   "tableView": false
                 },
                 {
-                  "input": true,
+                  "label": "studyEndDateIsPastApplicationException",
                   "tableView": false,
+                  "validateWhenHidden": false,
                   "key": "studyEndDateIsPastApplicationException",
-                  "label": "Application Exception - Study end date is past",
-                  "type": "hidden",
-                  "calculateValue": "value = \"studentApplicationException\";",
-                  "lockKey": true,
-                  "customConditional": "show = data.studyEndDateBeforeSixWeeksFromToday || data.selectedStudyEndDateBeforeSixWeeksFromToday;",
-                  "calculateServer": true
+                  "customConditional": "show = !data.isProgramSectionReadOnly && !!data.selectedStudyEndDateBeforeSixWeeksFromToday;",
```

Comment:
Is it required to bring `isProgramSectionReadOnly` here for the exception?

Reply:
Keeping the same logic from before. No plans for changing during this ticket.

Reply:
I ended up copying the logic from the warning. I will keep the one from the hidden as suggested.

Reply:
Reverted to the hidden field logic.